includes/db.php

```diff
@@ -76,16 +76,26 @@ class DB {
 	 *
 	 * Retrieve editor data from the database.
 	 *
-	 * @since 1.0.0
+	 * @param int $post_id Post ID.
+	 * @param string $status Optional. Post status. Default is `publish`.
 	 *
-	 * @access public
+	 * @return array Editor data.
+	 * @deprecated 3.1.0 Use `Plugin::$instance->documents->get( $post_id )->get_elements_raw_data( null, true )` instead
 	 *
-	 * @param int     $post_id           Post ID.
-	 * @param string  $status            Optional. Post status. Default is `publish`.
+	 * @access public
 	 *
-	 * @return array Editor data.
+	 * @since 1.0.0
 	 */
 	public function get_builder( $post_id, $status = self::STATUS_PUBLISH ) {
+		Plugin::$instance->modules_manager
+			->get_modules( 'dev-tools' )
+			->deprecation
+			->deprecated_function(
+				__METHOD__,
+				'3.1.0',
+				'Plugin::$instance->documents->get( $post_id )->get_elements_raw_data( null, true )'
+			);
+
```

Comment:
It needs to return `Plugin::$instance->documents->get( $post_id )->get_elements_raw_data( null, true )`
But, what will happen with the `status` args? (for 3rd users that uses it)